From c41dff1a5b9bf6627497550aa1e144c9e610516b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <cynerd@email.cz>
Date: Thu, 14 Oct 2021 15:24:27 +0200
Subject: [PATCH] pkglists/drivers: fix missing package 
 ath10k-firmware-qca988x-ct-htt

The package in question was renamed in upstream commit:
2e5e9b459ed53b42f8938294a98e2546be976c85
---
 pkglists/drivers.lua.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkglists/drivers.lua.m4 b/pkglists/drivers.lua.m4
index 451cf77..9bdf2e8 100644
--- a/pkglists/drivers.lua.m4
+++ b/pkglists/drivers.lua.m4
@@ -5,7 +5,7 @@ if options then
 if options.ath10k_ct or options.ath10k_ct_htt then
 	Install("kmod-ath10k-ct", { priority = 40 })
 	if options.ath10k_ct_htt then
-		Install("ath10k-firmware-qca988x-ct-htt", { priority = 40 })
+		Install("ath10k-firmware-qca988x-ct-full-htt", { priority = 40 })
 	else
 		Install("ath10k-firmware-qca988x-ct", { priority = 40 })
 	end
-- 
2.33.1

